README.md

```diff
@@ -7,3 +7,84 @@ Blog technique de la communauté NaN, disponible à cette adresse:
+
+   1. Chercher le package ruby avec msys2
+   ```
+   winget search ruby
+   ```
+
+   2. Trouver une version avec noté "Ruby 3.2 with MSYS2" dans la sortie de la commande précédente
+   exemple:
+   ```
+    Name                                   Id                                   Version      Match     Source
+    ----------------------------------------------------------------------------------------------------------
+    Ruby On Rails-simpleNeasyApp by WAGmob 9WZDNCRDN7Q9                         Unknown                msstore
+    Ruby Formatter                         9NN0RJL0D2NK                         Unknown                msstore
+    InstantRuby                            9WZDNCRDC1W3                         Unknown                msstore
+    RubyGo                                 9P55T10JZJBC                         Unknown                msstore
+    Sonic Pi                               SonicPi.SonicPi                      3.3.1        Tag: ruby winget
+    Ruby 3.1 with MSYS2                    RubyInstallerTeam.RubyWithDevKit.3.1 3.1.2-1      Tag: ruby winget
+    Ruby 3.0 with MSYS2                    RubyInstallerTeam.RubyWithDevKit.3.0 3.0.4-1      Tag: ruby winget
+    Ruby 2.7 with MSYS2                    RubyInstallerTeam.RubyWithDevKit.2.7 2.7.6-1      Tag: ruby winget
+    Ruby 2.6 with MSYS2                    RubyInstallerTeam.RubyWithDevKit.2.6 2.6.10-1     Tag: ruby winget
+    Ruby 3.1                               RubyInstallerTeam.Ruby.3.1           3.1.2-1      Tag: ruby winget
+    Ruby 3.0                               RubyInstallerTeam.Ruby.3.0           3.0.4-1      Tag: ruby winget
+    Ruby 2.7                               RubyInstallerTeam.Ruby.2.7           2.7.6-1      Tag: ruby winget
+    Ruby 2.6                               RubyInstallerTeam.Ruby.2.6           2.6.10-1     Tag: ruby winget
+    MAMP & MAMP PRO                        MAMP.MAMP                            4.2.0        Tag: ruby winget
+    Laragon                                LeNgocKhoa.Laragon                   5.0.0.210523 Tag: ruby winget
+    RubyMine (EAP)                         JetBrains.RubyMine.EarlyAccess       231.6890.13  Tag: ruby winget
+    RubyMine                               JetBrains.RubyMine                   2022.3.2     Tag: ruby winget
+    EditPlus                               ES-Computing.EditPlus                5.6          Tag: ruby winget
+    Ruby 3.2 with MSYS2                    RubyInstallerTeam.RubyWithDevKit.3.2 3.2.0-1      Tag: ruby winget
+   ```
+
+   3. Installer cette version en copiant collant l'Id de cette version
+   exemple :
+   ```
+   winget install RubyInstallerTeam.RubyWithDevKit.3.2
+   ```
+
+Vérifier que ruby est installé:
+
+```
+ruby -v
+```
+
+Cloner le dépôt github:
+
+```
+git clone https://github.com/NotANameServer/Not-a-Hub.git
+```
+
+Installer les dépendances:
+```
+bundle install
+```
+
+Build le site web:
+```
+bundle exec jekyll build
+```
```

Comment:
Ce sont ces étapes qui me font dire qu'on est plus dans un *how to contribute* qu'un *how to run*